duckduckgo/macos-browser
# Remove login import de-duplication feature flag

Task/Issue URL: https://app.asana.com/0/414235014887631/1207759292115042/f

**Description**:

For [✓ macOS: Deduplicate passwords on import](https://app.asana.com/0/0/1207597760316574), a feature flag was added to mitigate risk of negative user reaction / unforeseen issues. This should be removed once we're confident in the implementation.

The feature flag is named `deduplicateLoginsOnImport`

**Steps to test this PR**:
1. Check login import still deduplicates correctly (https://app.asana.com/0/72649045549333/1207597760316577/f)

**Definition of Done**:


---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
